docs/dev/zksync-v2/blocks-and-time.md

```diff
@@ -1,3 +1,24 @@
 # Block numbers and time
 
-TODO.
+On ZkSync there are two types of blocks: L2 blocks and L1 batches.
```

Comment:
Why do we need 2 types? Why the latter called "batches" and not "blocks" (though they're called blocks at the beginning of the sentence)?